Sources/Charcoal/CharcoalViewController.swift

```diff
@@ -32,6 +32,8 @@ public final class CharcoalViewController: UINavigationController {
         didSet { configure(with: filterContainer) }
     }
 
+    public var defaultMapMode: MapFilterMode = .radius
```

Comment:
I suggest that we turn this a computed property and override `didSet` to select the correct `selectedViewController` which makes it consistent with the rest of the interface; and will update the controller when the value changes as well.👍 

```suggestion
    public var defaultMapMode: MapFilterMode = {
        didSet {
            switch defaultMode {
                case .polygon:
                    selectedViewController = mapPolygonFilterViewController ?? mapRadiusFilterViewController
                case .radius:
                    selectedViewController = mapRadiusFilterViewController
         }
    }
}
```